docs/source/index.rst

```diff
@@ -0,0 +1,235 @@
+.. neo4j-genai-python documentation master file, created by
+   sphinx-quickstart on Tue Apr  9 16:36:43 2024.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Neo4j GenAI package for Python
+==============================
+
+This repository contains the official Neo4j GenAI features for Python.
+
+The purpose of this package is to provide a first party package to developers,
+where Neo4j can guarantee long term commitment and maintenance as well as being
+fast to ship new features and high performing patterns and methods.
+
+******
+Topics
+******
+
++ :ref:`api-documentation`
++ :ref:`types-documentation`
+
+.. toctree::
+    :maxdepth: 3
+    :caption: Contents:
+    :hidden:
+
+    api.rst
+    types.rst
+
+
+Usage
+=====
+
+************
+Installation
+************
+
+This package requires Python (>=3.8.1).
+
+To install the latest stable version, use:
+
+.. code:: bash
+
+    pip install neo4j-genai
+
+
+.. note::
+
+   It is always recommended to install python packages for user space in a virtual environment.
+
+********
+Examples
+********
+
+While the library has more retrievers than shown here, the following examples should be able to get you started.
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Performing a similarity search
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    from neo4j import GraphDatabase
+    from neo4j_genai import VectorRetriever
+
+    URI = "neo4j://localhost:7687"
+    AUTH = ("neo4j", "password")
+
+    INDEX_NAME = "embedding-name"
+
+    # Connect to Neo4j database
+    driver = GraphDatabase.driver(URI, auth=AUTH)
+
+    # Initialize the retriever
+    retriever = VectorRetriever(driver, INDEX_NAME)
+
+    # Run the similarity search
+    query_text = "How do I do similarity search in Neo4j?"
+    response = retriever.search(query_text=query_text, top_k=5)
```

Comment:
This is missing an embedder, copy what's in the README, I've updated that example a few weeks back.